src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -0,0 +1,295 @@
+import React, {
+    useState,
+    useEffect,
+    useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import Classnames from 'classnames';
+import Typography from '../../dataDisplay/typography';
+import Icon from '../../dataDisplay/icon';
+import ToolTip from '../../dataDisplay/tooltip';
+import makeStyles from '../../styles/makeStyles';
+import { BEM_NAVIGATION_BREADCRUMBS } from '../../global/constants';
+import {
+    routesToArray,
+    getPathNameBreadcrumbs,
+} from './breadcrumbsUtils';
+
+const propTypes = {
+    /**
+     * Instance of react-router core,
+     * breadcrums will be generated automatically using the routes object structure,
+     * location and push function.
+     */
+    router: PropTypes.shape({}).isRequired,
+    /**
+     * When false, is going to show only the previous route from the current location
```

Comment:
This comment is confusing.  It seems to be saying that "if a prop called `showOnlyPreviousRoute` is `false` then we are only going to show the previous route."  Huh?  That sounds like I would expect that to happen when it is _true_.

Please explain what this does both when it is true and when it is false.

Reply:
Is it something like this?
`showOnlyPreviousRoute === false` (default):
Foo > Bar > Baz > Quux

`showOnlyPreviousRoute === true`:
Baz > Quuz

I re-read the unit tests, and I think this is right...?

Reply:
You're right, I updated the comment in `breadcrumbs.jsx`, let me know if I can be way clearer with the prop description, thanks.

Reply:
The update looks good.  Thanks.
https://github.com/saddlebackdev/react-cm-ui/blob/507e4b80860a402c58eabfb8463b2fa56064307a/src/navigation/breadcrumbs/breadcrumbs.jsx#L27-L30

I'll resolve this.